PetGame/Services/PetService.cs

```diff
@@ -200,5 +221,267 @@ public bool DeletePet(ulong petId, ulong ownerId)
+
+            if (Activities.Count == 0)
+            {
+                hungerPercentage -= (HungerDecreasePerHour * HoursToCheck);
+            }
+            else
+            {
+                //check the type of activity and subtract percentage accordingly
+                foreach (Activity Activity in PastActivities)
+                {
+                    //if a Feeding activity has occured, increase hunger
+                    if (Activity.Type.Equals(ActivityType.Feeding))
+                    {
+                        hungerPercentage = ModifyPercentage(hungerPercentage, FeedingIncrease, PercentageOperation.Add);
+                    }
+                    //if a Training event has occured, subtract
+                    else if (Activity.Type.Equals(ActivityType.Training))
+                    {
+                        hungerPercentage = ModifyPercentage(hungerPercentage, TrainingDecrease, PercentageOperation.Subtract);
+                    }
+                    //if a Race event has occurred
+                    else if (Activity.Type.Equals(ActivityType.Race))
+                    {
+                        hungerPercentage = ModifyPercentage(hungerPercentage, RaceDecrease, PercentageOperation.Subtract);
+                    }
+                    //else, decrease hunger
+                    else
+                    {
+                        hungerPercentage = ModifyPercentage(hungerPercentage, HungerDecreasePerHour, PercentageOperation.Subtract);
+                    }
+                }
+            }
+            return hungerPercentage;
+        }
+
+        /// <summary>
+        /// Either adds to or subtracts from the passed in percentage by the supplied changeFactor,
+        /// based on the specified operation
+        /// </summary>
+        /// <param name="percentage">The percentage to modify</param>
+        /// <param name="changeFactor">The about to add or subtract</param>
+        /// <param name="operation">Which operation to perform (add or subtract)</param>
+        /// <returns>The new percentage as a double</returns>
+        private double ModifyPercentage(double percentage, double changeFactor, PercentageOperation operation)
+        {
+            //a stands for add
+            if (operation.Equals(PercentageOperation.Add))
+            {
+                //percentage is currently less than the max, and after
+                //the operation, will still be less than the max
+                if (percentage < MaximumPercentage && ((percentage += changeFactor) <= MaximumPercentage))
+                {
+                    percentage += changeFactor;
+                    return percentage;
+                }
+                //return the max if the operation would exceed it
+                else
+                {
+                    return MaximumPercentage;
+                }
+            }
+            //s stands for subtract
+            else if (operation.Equals(PercentageOperation.Subtract))
```

Comment:
`enum`s are primitives, they are backed by only primitive types. You do not need to use `Equals()`